jwst/cube_build/ifu_cube.py

```diff
@@ -538,14 +545,23 @@ def build_ifucube(self):
                 if self.interpolation == 'pointcloud':
                     t0 = time.time()
                     pixelresult = self.map_detector_to_outputframe(this_par1,
-                                                                   this_par2,
                                                                    subtract_background,
                                                                    ifile)
 
-                    coord1, coord2, wave, flux, rois_pixel, roiw_pixel, weight_pixel,\
+                    coord1, coord2, wave, flux, slice, rois_pixel, roiw_pixel, weight_pixel,\
                         softrad_pixel, alpha_det, beta_det = pixelresult
                     t1 = time.time()
                     log.info("Time to transform pixels to output frame = %.1f.s" % (t1 - t0,))
+
+# If setting the DQ plane of the IFU
```

Comment:
pep8/flake8 won't like this comment starting way on the left edge. Should indent to match the code block.

Reply:
ok